### PR TITLE
task_1, task_2 

### PR DESCRIPTION
**Pull request by anetso18**

### CODE REVIEW

File: exercise_1/task_1.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#include <cctype>
+
+using namespace std;
+
+int main()
+{
+	string words;	
+	int vowels = 0, consonants = 0, others = 0;
+	
+	cout << "Enter words (q to quit)\n";
```
Comment (reczepio) [inline exercise_1/task_1.cpp]: If you want to be sure, the note is displayed, please use "std::endl" instead of "\n"

File: exercise_1/task_1.cpp [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+				consonants++;
+				break;
+			}
+		}
+		else {
+			others++;
+		}
+	}
+  
+	cout << vowels << " words beginning with vowel\n";
```
Comment (reczepio) [inline exercise_1/task_1.cpp]: Same comment:
rather "std::endl" then "\n"

File: exercise_1/task_1.cpp [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+		if (isalpha(words[0]))
+		{
+			switch (tolower(words[0])) {
+			case 'a':
+			case 'i':
+			case 'e':
+			case 'o':
+			case 'u':
+			case 'y':
+				vowels++;
```
Comment (reczepio) [inline exercise_1/task_1.cpp]: Rather ++vowels then vowels++. 
++vowels is more efficient.

File: exercise_1/task_1.cpp [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+		}
+		else {
+			others++;
+		}
+	}
+  
+	cout << vowels << " words beginning with vowel\n";
+	cout << consonants << " words beginning with consonant\n";
+	cout << others << " others";
+}
```
Comment (pstanisz) [inline exercise_1/task_1.cpp:38]: main() is probably the only non-void function in C++ that is not complaining about missing return statement.
From C++14 standard:
> A return statement in main has the effect of leaving the main function (destroying any objects with automatic
> storage duration) and calling std::exit with the return value as the argument. If control reaches the end
> of main without encountering a return statement, the effect is that of executing
> return 0;

However, it is good to remember about `return 0;` or better `return EXIT_SUCCESS;` at the end of main function.
Why EXIT_SUCCESS or EXIT_FAILURE is better, then just returning '0' or '1'? Portability reasons :)

**Review by reczepio — COMMENTED** [review]
+1

**Comment by pstanisz** [comment]
+2

File: exercise_1/task_1.cpp (hunk 4 above)
Comment (anetso18) [reply exercise_1/task_1.cpp:38]: Hi, when should I merge a pull request?

Comment (anetso18) [reply exercise_1/task_1.cpp:38]: I've also added one commit within this pull request below and not sure if you saw it.

**Review by pstanisz — APPROVED** [review]

**Pull request merged**